ingress/controllers/nginx/README.md

```diff
@@ -6,15 +6,28 @@ This is a nginx Ingress controller that uses [ConfigMap](https://github.com/kube
 ## What it provides?
```

Comment:
suggest completely removing this. It's taking up too much space too hing on an important doc.